feat: performance results file validation (#216)
callstack/reassure: .changeset/angry-moles-bake.md

---

> '@callstack/reassure-compare': minor

Reviewer: This is also minor version bump for the main package and cli
```suggestion
'reassure': minor
'@callstack/reassure-cli': minor
'@callstack/reassure-compare': minor
```

---

> Validate performance results files input to compare package

Reviewer: ```suggestion
Validate format of performance results files when loading for comparison
```
